src/renderer/appBadge.ts

```diff
@@ -24,7 +22,8 @@ export function setBadge() {
         let totalCount = mentionCount + pendingRequests;
         if (!totalCount && hasUnread && !disableUnreadBadge) totalCount = -1;
 
-        VesktopNative.app.setBadgeCount(totalCount);
+        if (Settings.store.appBadge || Settings.store.trayBadge)
+            VesktopNative.app.setBadgeCount(totalCount, Settings.store.appBadge, Settings.store.trayBadge);
```

Comment:
what's the point of passing these settings via ipc? the main process also has access to these settings so you don't need to do that

Reply:
I didn't know that- will change

Reply:
you can import the settings api from src/main/settings.ts